extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -429,11 +436,49 @@ export class BookTocManager implements IBookTocManager {
 		}
 	}
 
+	public async addNewNotebook(notebookName: string, notebookPath: string, fileType: FileType, book?: BookModel): Promise<void> {
+		if (fileType === FileType.Notebook) {
+			await this.createNotebook(notebookPath.concat(fileType), undefined);
+		} else {
+			await fs.writeFile(notebookPath.concat(fileType), '');
+		}
+		if (book) {
+			let toc = yaml.safeLoad((await fs.readFile(book.tableOfContentsPath, 'utf8')));
+			let notebookToc: JupyterBookSection = {
+				title: notebookName,
+				file: path.posix.join(path.posix.sep, notebookName)
+			};
+			if (book.version === BookVersion.v1) {
+				notebookToc = convertTo(book.version, notebookToc);
+			}
+			toc.push(notebookToc);
+			await fs.writeFile(book.tableOfContentsPath, yaml.safeDump(toc, { lineWidth: Infinity, noRefs: true, skipInvalid: true }));
+		} else {
+			vscode.commands.executeCommand('bookTreeView.openBook', notebookPath.concat(fileType), false, notebookPath.concat(fileType), true);
```

Comment:
So we open the file if it's not being added to a book? I would think we'd want to open the file regardless